site/content/en/docs/administration/advanced/upgrade_guide.md

```diff
@@ -56,6 +56,52 @@ To upgrade CVAT, follow these steps:
   docker logs cvat_server -f
   ```
 
+## Upgrade CVAT after v2.25.0
+
+In version 2.25.0, CVAT changed the location where the export cache is stored.
+The following Python script can be used to remove outdated files from the previous location:
+
+```python
+import shutil
+from pathlib import Path
+from django.utils import timezone
+from tqdm import tqdm
+from cvat.apps.engine.models import Job, Project, Task
+
+
+def cleanup_outdated_cache():
+   now = timezone.now()
+
+   for Model in (Project, Task, Job):
+      print(f"Deleting the export cache for {Model.__name__.lower()}s...")
+      queryset = Model.objects.filter(created_date__lt=now)
+      objects_count = queryset.count()
+      if objects_count < 1:
+         continue
+
+      print(f"The {objects_count} folder{'s' if objects_count > 1 else ''} are going to be checked")
+
+      for obj in tqdm(queryset.iterator(), total=objects_count):
+         export_cache_dir = Path(obj.get_dirname()) / "export_cache"
+         if export_cache_dir.exists():
+               shutil.rmtree(export_cache_dir)
+
+
+if __name__ == "__main__":
+    cleanup_outdated_cache()
+
+```
+
+### How to run the script
+
+1. Save the script as `cleanup_script.py` in a directory where `manage.py` is located
+1. Run Django shell command: `python manage.py shell`
```

Comment:
To run such one-time scripts, we probably need to add in the future a custom management command like `runscript` from the `django-extensions` package.